recipes/update-host-based-configurations.rb

```diff
@@ -7,6 +7,7 @@
 opencast_repo_root = node[:opencast_repo_root]
 
 production_deploy_root = opencast_repo_root + '/current'
+stack_name = stack_shortname
```

Comment:
Is stack_shortname already a param somewhere?

Reply:
It's a function in libraries/defaults.rb.